### ISSUE TITLE
Diameter symbol

### ISSUE THREAD

**Issue by bisato** [ForNeVeR/wpf-math#247]
Hello,

is it possible to draw a diameter symbol like in TeX? \diameter

Greetings

Tobias

**Comment by ForNeVeR** [comment]
Hello! It looks like this character is absent from the Computer Modern font we use. So, right now the answer is: not.

I was going to suggest you use `\text{∅}` as a substitute, but it's [failed](https://github.com/ForNeVeR/wpf-math/issues/248), we'll investigate.

As a bigger answer: we have plans to add more fonts and more characters available by default, but for now this is blocked on #107.

**Comment by rstm-sf** [comment]
Hello!

Are you comfortable with this `\text{Ø}`?
> [LATIN CAPITAL LETTER O WITH STROKE](https://www.woutware.com/Forum/Topic/759)

**Comment by bisato** [comment]
Thank you rstm-sf that would be a good solution

**Comment by HolzetheKid** [comment]
Hallo,

the current release is supporting the diameter symbol.
![image](https://user-images.githubusercontent.com/9831731/174489059-2249b92d-3165-47b1-b93b-eb03b8d0fd06.png)

So i would suggest that this issue is closed.

**Comment by ForNeVeR** [comment]
Yes, `\text{Ø}` works (according to #107 which is marked as fixed), but `\diameter` still isn't.
